lighthouse/classes/messages/sequencescape_messages.py

```diff
@@ -0,0 +1,73 @@
+from flask import current_app as app
+from lighthouse.messages.message import Message
+from typing import Dict
+import requests
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class SequencescapeMessage(Message):
+    def __init__(self):
+        self._barcode = None
+        self._contents = {}
+
+    def set_barcode(self, barcode):
+        self._barcode = barcode
+
+    def set_well_sample(self, location, sample_info):
+        self._contents[location] = {"content": sample_info}
+
+    def render(self) -> Message:
+        message_content = self.construct_sequencescape_message(
+            barcode=self._barcode,
+            purpose_uuid=app.config["SS_UUID_PLATE_PURPOSE_CHERRYPICKED"],
+            study_uuid=app.config["SS_UUID_STUDY_CHERRYPICKED"],
+            wells=self._contents,
+            events=[],
+        )
+
+        return Message(message_content)
+
+    def construct_sequencescape_message(self, barcode, purpose_uuid, study_uuid, wells, events):
+        body = {
+            "barcode": barcode,
+            "purpose_uuid": purpose_uuid,
+            "study_uuid": study_uuid,
+            "wells": wells,
+            "events": events,
+        }
+
+        return {"data": {"type": "plates", "attributes": body}}
+
+    def send_to_ss(self):
+        message = self.render()
+        return self._send_to_ss(
+            ss_url=app.config["SS_PLATE_CREATION_ENDPOINT"],
+            headers={"X-Sequencescape-Client-Id": app.config["SS_API_KEY"], "Content-type": "application/json"},
+            data=message.payload(),
+        )
+
+    def _send_to_ss(self, ss_url: str, headers: Dict[str, str], data: str) -> requests.Response:
```

Comment:
This is a static method. It's also a general method so might be worth moving it to a general class?